bentoml/_internal/utils/analytics/usage_stats.py

```diff
@@ -36,9 +38,11 @@
 
 BENTOML_DO_NOT_TRACK = "BENTOML_DO_NOT_TRACK"
 USAGE_TRACKING_URL = "https://t.bentoml.com"
-SERVE_USAGE_TRACKING_INTERVAL_SECONDS = int(12 * 60 * 60)  # every 12 hours
 USAGE_REQUEST_TIMEOUT_SECONDS = 1
 
+# Send a serve event every 12 hours
+_serve_usage_tracking_interval_seconds = int(12 * 60 * 60)
```

Comment:
why renaming? use all cap for constants

Reply:
oops

Reply:
I think my refactoring library have some issues